api/resources/views/admin/groups/edit.blade.php

```diff
@@ -0,0 +1,44 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="container">
+        <div class="card">
+            <div class="card-body">
+                <div class="form-group row">
+                    <label class="col-md-4 col-form-label text-md-right">Current group</label>
+
+                    <div class="col-md-6">
+                        <label class="col-form-label">IFF-STUDENTAS</label>
+                    </div>
+                </div>
+                <form method="POST" class="align-content-center">
+                    @csrf
+                    <div class="form-group row">
+                        <label for="group" class="col-md-4 col-form-label text-md-right">New group</label>
+
+                        <div class="col-md-6">
+                            <select name="group">
+                                <option>IFF-1</option>
+                                <option>IFF-2</option>
+                                <option>IFF-3</option>
```

Comment:
two same choices "IFF-3" and what if i would like to chose IFF-7? or IFF-9? Or am i wrong and it is only available to choice 4 group names?

Reply:
This is just a placeholder for presentational purposes.